burgle.lic

```diff
@@ -151,6 +415,7 @@ class Burgle
     end
   end
 
+  #ripped out of steal.lic
```

Comment:
❔  Do we need a `common-thieving` or `common-stealing` (or you pick the name)?

Reply:
While that comes out of steal.lic, I would think it would be better in common-items.  It's really just putting an item away or dropping it if unsuccessful.

Reply:
Sounds reasonable to me. Not necessary for this PR, mind you, just something to consider for the future.